src/components/WorkPoolQueueCreateForm.vue

```diff
@@ -63,7 +63,7 @@
   const isRequired: ValidationRule<string | undefined> = (value) => value !== undefined && value.trim().length > 0
 
   const isGreaterThanZero: ValidationRule<number | undefined> = (value, name) => {
-    if (value && value > 0) {
+    if (value == null || value > 0) {
```

Comment:
It looks like this field also is using `isRequired` rule? These seem to conflict and will probably lead to unexpected results. 

I would also like to see a change to rule name to match updated logic.

Reply:
I changed the name - this field didn't use the `isRequired` rule, only the `name` field was using that